etl/etl/utils.py

```diff
@@ -121,3 +121,27 @@ def commaize_number(number):
         return retn
     except Exception as e:
         return number
+
+
+def parse_pare(pare_option):
+    """Parses the PARE_OPTION and returns a tuple of
+    (PARE_FACTOR, KEYS_TO_INCLUDE) where PARE_FACTOR is an int
+    by which proposals should be reduced, and KEYS_TO_INCLUDE
+    is a list of proposals to include by key.  Only one of the
+    two will ever be returned."""
```

Comment:
No, they'll both be returned, it's just that at least one of them will be None :-).  (But seriously, it's important to be specific about that, since it's the calling discipline and in fact you do depend on the "is not None" test in the caller.)